### ISSUE TITLE
V0.2.4b Will Not Run on Windows 10

### ISSUE THREAD

**Issue by 2braincells2go** [jyapayne/Web2Executable#79]
I have tried to get 0.2.4b standalone and also installer version to run on Windows 10. Nothing happens.

Does not start and does not appear when viewing programs in task manager. Clicking on exe has no effect at all and does not create the downloads directory.

There are no errors, no error logs. Running as administrator has no effect other than error about unsafe since exe not signed. Using the "Program Compatibility Troubleshooter" did not solve issue either, tried many different combinations without success.

v0.2.3b starts up without issues on Win 10. Having some problems with it on Win 10 but it does start:)

Thought you may like to know since Win 10 is headed towards us like run-a-way train with no way to stop it, lol

If you need me to test anything, do not hesitate to ask Joey!

**Comment by jyapayne** [comment]
Alright :) Thanks for filing the issue. I'll see what I can scrounge up for Windows 10.

**Comment by jyapayne** [comment]
I installed Windows 10 and Web2Exe 0.2.4b works as expected :o The question is why is yours not working?

Is there anything in the error log located here: C:\Users<username>\AppData\Local\Web2Executable\Web2Executable\files\error.log?

**Comment by 2braincells2go** [comment]
Very interesting. I had looked for the log file when trying to figure out what was happening. Did not even see Web2Executable directory in C:\Users\AppData\Local\. Many others of course but no Web2Exe.

I have another Win 10 machine and will attempt to run on it. Late here so will report back in the morning.

Thanks Joey.

**Comment by 2braincells2go** [comment]
There has to be something in common with my two Win 10 installs. Neither will run Web2Exe 0.2.4b.

First thought it might be my WebRoot SecureAnywhere blocking, but second Win 10 machine tried does not have WebRoot installed and it also has issues with starting Web2Exe 0.2.4b.

Both machines started running the Win 10 preview way back and were upgraded to general release month or so ago. Wondering if this has something to do with my issues, I am assuming you installed new general public release of 10?

Bottom line, it appears to be my issue so will close this. Will let you know in future if I figure it out.

Thanks much Joey.

**Comment by jyapayne** [comment]
Yeah, I installed the public release using the [media creation tool](http://windows.microsoft.com/en-ca/windows-10/media-creation-tool-install).

It's weird that it doesn't work. Do let me know if you figure out what's up :)

**Comment by 2braincells2go** [comment]
Hi Joey. Not sure if this will help as V0.2.4b is working for you but jsut in case here is what I discovered:

Finally got error message to popup after many times of deinstall/reinstall. The message is releated to this article:

http://ntcoder.com/bab/2014/02/27/troubleshooting-side-by-side-issues-using-sxstrace/

My logfile is pasted below. Far from Windows guru so not sure if this was caused by starting with Win 10 Technical Preview, very possbile.

Got to leave for the day, check back in with you this evening.

Logfile:

=================
Begin Activation Context Generation.
Input Parameter:
	Flags = 0
	ProcessorArchitecture = Wow32
	CultureFallBacks = en-US;en
	ManifestPath = C:\Program Files (x86)\Web2Exe\Web2Exe.exe
	AssemblyDirectory = C:\Program Files (x86)\Web2Exe\
	Application Config File = 
-----------------
INFO: Parsing Manifest File C:\Program Files (x86)\Web2Exe\Web2Exe.exe.
	INFO: Manifest Definition Identity is Web2Exe,processorArchitecture="x86",type="win32",version="1.0.0.0".
	INFO: Reference: Microsoft.VC90.CRT,processorArchitecture="x86",publicKeyToken="1fc8b3b9a1e18e3b",type="win32",version="9.0.21022.8"
	INFO: Reference: Microsoft.VC90.MFC,processorArchitecture="x86",publicKeyToken="1fc8b3b9a1e18e3b",type="win32",version="9.0.21022.8"
	INFO: Reference: Microsoft.Windows.Common-Controls,language="&#x2a;",processorArchitecture="x86",publicKeyToken="6595b64144ccf1df",type="win32",version="6.0.0.0"
INFO: Resolving reference Microsoft.VC90.CRT,processorArchitecture="x86",publicKeyToken="1fc8b3b9a1e18e3b",type="win32",version="9.0.21022.8".
	INFO: Resolving reference for ProcessorArchitecture WOW64.
		INFO: Resolving reference for culture Neutral.
			INFO: Applying Binding Policy.
				INFO: No publisher policy found.
				INFO: No binding policy redirect found.
			INFO: Begin assembly probing.
				INFO: Did not find the assembly in WinSxS.
				INFO: Attempt to probe manifest at C:\WINDOWS\assembly\GAC_32\Microsoft.VC90.CRT\9.0.21022.8__1fc8b3b9a1e18e3b\Microsoft.VC90.CRT.DLL.
				INFO: Did not find manifest for culture Neutral.
			INFO: End assembly probing.
	INFO: Resolving reference for ProcessorArchitecture x86.
		INFO: Resolving reference for culture Neutral.
			INFO: Applying Binding Policy.
				INFO: Find publisher policy at C:\WINDOWS\WinSxS\manifests\x86_policy.9.0.microsoft.vc90.crt_1fc8b3b9a1e18e3b_9.0.30729.9158_none_f47f259ef656232a.manifest
				INFO: Publisher Policy redirected assembly version.
				INFO: Post policy assembly identity is Microsoft.VC90.CRT,processorArchitecture="x86",publicKeyToken="1fc8b3b9a1e18e3b",type="win32",version="9.0.30729.9158".
			INFO: Begin assembly probing.
				INFO: Attempt to probe manifest at C:\WINDOWS\WinSxS\manifests\x86_microsoft.vc90.crt_1fc8b3b9a1e18e3b_9.0.30729.9158_none_5091b51ebcb97cdc.manifest.
				INFO: Manifest found at C:\WINDOWS\WinSxS\manifests\x86_microsoft.vc90.crt_1fc8b3b9a1e18e3b_9.0.30729.9158_none_5091b51ebcb97cdc.manifest.
			INFO: End assembly probing.
INFO: Resolving reference Microsoft.VC90.CRT.mui,language="&#x2a;",processorArchitecture="x86",publicKeyToken="1fc8b3b9a1e18e3b",type="win32",version="9.0.30729.9158".
	INFO: Resolving reference for ProcessorArchitecture WOW64.
		INFO: Resolving reference for culture en-US.
			INFO: Applying Binding Policy.
				INFO: No publisher policy found.
				INFO: No binding policy redirect found.
			INFO: Begin assembly probing.
				INFO: Did not find the assembly in WinSxS.
				INFO: Attempt to probe manifest at C:\WINDOWS\assembly\GAC_32\Microsoft.VC90.CRT.mui\9.0.30729.9158_en-US_1fc8b3b9a1e18e3b\Microsoft.VC90.CRT.mui.DLL.
				INFO: Did not find manifest for culture en-US.
			INFO: End assembly probing.
		INFO: Resolving reference for culture en.
			INFO: Applying Binding Policy.
				INFO: No publisher policy found.
				INFO: No binding policy redirect found.
			INFO: Begin assembly probing.
				INFO: Did not find the assembly in WinSxS.
				INFO: Attempt to probe manifest at C:\WINDOWS\assembly\GAC_32\Microsoft.VC90.CRT.mui\9.0.30729.9158_en_1fc8b3b9a1e18e3b\Microsoft.VC90.CRT.mui.DLL.
				INFO: Did not find manifest for culture en.
			INFO: End assembly probing.
	INFO: Resolving reference for ProcessorArchitecture x86.
		INFO: Resolving reference for culture en-US.
			INFO: Applying Binding Policy.
				INFO: No publisher policy found.
				INFO: No binding policy redirect found.
			INFO: Begin assembly probing.
				INFO: Did not find the assembly in WinSxS.
				INFO: Attempt to probe manifest at C:\WINDOWS\assembly\GAC_32\Microsoft.VC90.CRT.mui\9.0.30729.9158_en-US_1fc8b3b9a1e18e3b\Microsoft.VC90.CRT.mui.DLL.
				INFO: Did not find manifest for culture en-US.
			INFO: End assembly probing.
		INFO: Resolving reference for culture en.
			INFO: Applying Binding Policy.
				INFO: No publisher policy found.
				INFO: No binding policy redirect found.
			INFO: Begin assembly probing.
				INFO: Did not find the assembly in WinSxS.
				INFO: Attempt to probe manifest at C:\WINDOWS\assembly\GAC_32\Microsoft.VC90.CRT.mui\9.0.30729.9158_en_1fc8b3b9a1e18e3b\Microsoft.VC90.CRT.mui.DLL.
				INFO: Did not find manifest for culture en.
			INFO: End assembly probing.
INFO: Resolving reference Microsoft.VC90.MFC,processorArchitecture="x86",publicKeyToken="1fc8b3b9a1e18e3b",type="win32",version="9.0.21022.8".
	INFO: Resolving reference for ProcessorArchitecture WOW64.
		INFO: Resolving reference for culture Neutral.
			INFO: Applying Binding Policy.
				INFO: No publisher policy found.
				INFO: No binding policy redirect found.
			INFO: Begin assembly probing.
				INFO: Did not find the assembly in WinSxS.
				INFO: Attempt to probe manifest at C:\WINDOWS\assembly\GAC_32\Microsoft.VC90.MFC\9.0.21022.8__1fc8b3b9a1e18e3b\Microsoft.VC90.MFC.DLL.
				INFO: Did not find manifest for culture Neutral.
			INFO: End assembly probing.
	INFO: Resolving reference for ProcessorArchitecture x86.
		INFO: Resolving reference for culture Neutral.
			INFO: Applying Binding Policy.
				INFO: No publisher policy found.
				INFO: No binding policy redirect found.
			INFO: Begin assembly probing.
				INFO: Did not find the assembly in WinSxS.
				INFO: Attempt to probe manifest at C:\WINDOWS\assembly\GAC_32\Microsoft.VC90.MFC\9.0.21022.8__1fc8b3b9a1e18e3b\Microsoft.VC90.MFC.DLL.
				INFO: Attempt to probe manifest at C:\Program Files (x86)\Web2Exe\Microsoft.VC90.MFC.DLL.
				INFO: Attempt to probe manifest at C:\Program Files (x86)\Web2Exe\Microsoft.VC90.MFC.MANIFEST.
				INFO: Attempt to probe manifest at C:\Program Files (x86)\Web2Exe\Microsoft.VC90.MFC\Microsoft.VC90.MFC.DLL.
				INFO: Attempt to probe manifest at C:\Program Files (x86)\Web2Exe\Microsoft.VC90.MFC\Microsoft.VC90.MFC.MANIFEST.
				INFO: Did not find manifest for culture Neutral.
			INFO: End assembly probing.
	ERROR: Cannot resolve reference Microsoft.VC90.MFC,processorArchitecture="x86",publicKeyToken="1fc8b3b9a1e18e3b",type="win32",version="9.0.21022.8".
ERROR: Activation Context generation failed.
End Activation Context Generation.

**Comment by jyapayne** [comment]
Hmm. Unfortunately this doesn't help a whole lot. There's not really anything useful I can glean from this. Sorry :(

**Comment by 2braincells2go** [comment]
Missing file:

"So the above error means the application couldn’t find msvcr90d.dll in the above location. The way I would solve this is to create a setup project in VC9 and install the merge modules onto the target machine. Please note the dll’s are debug binaries else you could have just installed the redist’s."

I just made the directory manually and added the dll.

**Comment by jyapayne** [comment]
That's so weird. I didn't change anything in my windows build environment, so 0.2.4b should work just as well. I'll look into building it with a fresh Windows 7 install. Thank you very much for investigating!

**Comment by jyapayne** [comment]
Does this version work? I installed a fresh Windows 7 with only the necessary redistributable files.

http://www.mediafire.com/download/oyjo6gmgskgwkqy/Web2ExeWin-v0.2.5b.zip

**Comment by 2braincells2go** [comment]
Hi Joey. Sorry for delay, traveling and little time to be online.

I tried the version on my laptop and it worked! Did not before so big improvement for me. Maybe all this is because I started with the Win 10 technical preview? No clue but least now works.

Will try on my desktop when I return home tomorrow.

Thanks much.

**Comment by 2braincells2go** [comment]
Fixed issue I was having.
